### PR TITLE
Expanded participant section

### PR DESCRIPTION
**Pull request by TelegramSam**

### CODE REVIEW

**Comment by mccown** [comment]
This is a good PR.  I like the new organization / formatting.

**Comment by TelegramSam** [comment]
Fixes #34 
Fixes #35

File: credential-trust-establishment/spec.md [hunk 1]
```diff
@@ -96,6 +96,71 @@ TODO: Include a note about how to link to a specific version / general version o
...
+
+### Linked Goverance
+
+Linked governance files provide a flexible mechanism for governing larger ecosystems. Higher authorities can bundle governance together. Lower authorities can reference higher authorities. The result is a straightforward mecanism for practical ecosystem governance.
+
+Linked governance allows for governance chains to exist - the inclusion of an external governance file allows that governance to itself reference other governance files. While in theory this allows for infiniately long chains, in practice the requirement for the publisher to trust linked files places a reasonable limit. Also see (Expectations)[#expectations] for further commentary on the responsiblities of the publisher and reader.
+
+### Versioning
+
+Versioning is a way to allow users of decentralized ecosystem governance to track/use the current version of published governance or track/use older version of the file. This allowes flexibility of using/tracking most current version of DEGov file at any time.
```
[the remark this answers is not in the document]
Comment (Eldersonar) [reply credential-trust-establishment/spec.md:108]: version of  DEGov file at any time should be version of the DEGov file at any time?

File: credential-trust-establishment/spec.md [hunk 2]
```diff
@@ -224,68 +301,46 @@ The new section contains the enumerated sections of roles for a participant, alo
...
-
-```json
-{
-  "current_version": "https://example.com/path/to/cte/1.json"
-}
-```
-
-The base uri can point to a file with any file name of your choice as long as this is consistent within the ecosystem you are working with. Example:
+### Description
+This schema provides additional information for each identifier in the ecosystem. This is helpful for bootstrapping ecosystems without a reliable external system of participant identity. The folowing example shows names, email addresses, and the associated website for each participant.
```
Comment (Eldersonar) [inline credential-trust-establishment/spec.md:305]: "folowing" should be "following".

**Review by Eldersonar — CHANGES_REQUESTED** [review]
Minor typos to fix

File: credential-trust-establishment/spec.md [hunk 3]
```diff
@@ -96,6 +96,71 @@ TODO: Include a note about how to link to a specific version / general version o
 }
 ```
 
+
+### Linked Governance
+
+Linked governance files provide a flexible mechanism for governing larger ecosystems. Higher authorities can bundle governance together. Lower authorities can reference higher authorities. The result is a straightforward mechanism for practical ecosystem governance.
+
+Linked governance allows for governance chains to exist - the inclusion of an external governance file allows that governance to itself reference other governance files. While in theory this allows for infinitely long chains, in practice the requirement for the publisher to trust linked files places a reasonable limit. Also see (Requirements of Ecosystem Participants)[#requirements-of-ecosystem-participants] for further commentary on the responsiblities of the publisher and reader.
```
Comment (Eldersonar) [inline credential-trust-establishment/spec.md:104]: "responsiblities" should be "responsibilities".

File: credential-trust-establishment/spec.md [hunk 4]
```diff
@@ -176,7 +241,19 @@ This example shows potential roles used in a univeristy diploma ecosystem applic
 ...
 ```
 
-## Linking Participants to Roles
+## Participants
```
Comment (Eldersonar) [inline credential-trust-establishment/spec.md:244]: "univeristy" should be "university" in the section "Roles" section above

File: credential-trust-establishment/spec.md (hunk 3 above)
Comment (Eldersonar) [reply credential-trust-establishment/spec.md:104]: is linking correct? (Requirements of Ecosystem Participants)[#requirements-of-ecosystem-participants]

**Review by Eldersonar — CHANGES_REQUESTED** [review]
please, see more

**Review by Eldersonar — APPROVED** [review]
Thank you, looks good!

**Pull request merged**